odxtools/diaglayer.py

```diff
@@ -720,16 +717,13 @@ def get_doip_logical_ecu_address(self, protocol_name: Optional[str] = None) -> O
         return int(ecu_addr)
 
     def get_doip_logical_gateway_address(self,
-                                         protocol_name: Optional[str] = None,
-                                         is_functional: Optional[bool] = False) -> Optional[int]:
```

Comment:
Removing the `is_functional` parameter in any of these methods is a breaking change and should result in a new major release of odxtools if the removal is really necessary.

Reply:
This is still true IMO.